### PR TITLE
Use stable 6.0.300

### PR DESCRIPTION
**Pull request by inforithmics**
Maui Release Candidate 3 Updates

### CODE REVIEW

**Comment by inforithmics** [comment]
I guess appveyor fails so fast because it has still Visual studio 2022 17.1 installed instead of 17.2 (which github actions has installed).

**Comment by inforithmics** [comment]
I think with this pull request appveyor could be disabled. Because now the unit tests are run on windows and samples projects compile again. And maintaining two Build pipelines is overkill. and github actions builds it on mac too which appveyor does not.

**Comment by inforithmics** [comment]
1 Unit tests work locally but fail on github action I'll try to add an retry sttep.

**Comment by inforithmics** [comment]
appveyor build still fails don't know what happens there:
Main changes:
1) Introduced net6.0 as additional target for mapsui (reducing dependencies for packages that are included by default in 
net6.0)
2) added unit tests to github actions
3) compile the samples projects in github actions and repairing a lot of failures seems that appveyor didn't work properly to detect regressions there.
4) some samples are commented out they work in Visual Studio but not in github actions will reenable and fix them step by step. (Maybe in this pull request or in a later one, because now a lot more examples work now again).

**Comment by inforithmics** [comment]
All Samples build now on Windows github action. For passing all samples on Mac Github action the mac image needs to be updated to Visual Studio 2022 for mac.

**Comment by inforithmics** [comment]
I fixed now geotiff and wmts and wmtsmichelin sample

**Comment by inforithmics** [comment]
Fixed the Atlas Samples too

File: Samples/Mapsui.Samples.Uno/Mapsui.Samples.Uno.Mobile/Mapsui.Samples.Uno.Mobile.csproj [hunk 1]
```diff
@@ -49,4 +48,109 @@
     <ProjectReference Include="..\..\Mapsui.Samples.CustomWidget\Mapsui.Samples.CustomWidget.csproj" />
   </ItemGroup>
   <Import Project="..\Mapsui.Samples.Uno.Shared\Mapsui.Samples.Uno.Shared.projitems" Label="Shared" />
+  <ItemGroup>
```
Comment (pauldendulk) [inline Samples/Mapsui.Samples.Uno/Mapsui.Samples.Uno.Mobile/Mapsui.Samples.Uno.Mobile.csproj:51]: Please excluded the items in the list below by excluding the folder.

I am surprised they did show up at all, or perhaps was the intention to just delete them, and not exclude them from the csproj?

**Review by pauldendulk — COMMENTED** [review]
Magnificent. 

I have one remark you could look at after I merge.

File: Samples/Mapsui.Samples.Uno/Mapsui.Samples.Uno.Mobile/Mapsui.Samples.Uno.Mobile.csproj (hunk 1 above)
Comment (inforithmics) [reply Samples/Mapsui.Samples.Uno/Mapsui.Samples.Uno.Mobile/Mapsui.Samples.Uno.Mobile.csproj:51]: I'll do that

**Pull request merged**